mad-dog/src/helpers.js

```diff
@@ -97,27 +100,57 @@ const addUsersWithoutProfileImageOnSignUp = async (
 async function _addUsers ({ userCount, executeAll, executeOne, existingUserIds, addedUserIds, walletIndexToUserIdMap, uploadProfilePic = true }) {
   await logOps('Add users', async () => {
     const users = genRandomUsers(userCount)
+
     await executeAll(async (libs, i) => {
       try {
-      // If user already exists, do not recreate user and use existing user
+        // If user already exists, do not recreate user and use existing user
         const existingUser = await getUser({ executeOne, walletIndex: i })
         let userId
+
         if (existingUser) {
           logger.info(`Found existing user=${existingUser.user_id}`)
           existingUserIds.push(existingUser.user_id)
           userId = existingUser.user_id
+
         } else {
-          logger.info('Creating new user...')
-          const userMetadata = users[i]
-          let newUserId
-          if (uploadProfilePic) {
-            newUserId = await uploadProfileImagesAndAddUser(libs, userMetadata, USER_PIC_PATH)
-          } else {
-            newUserId = await addUser(libs, userMetadata)
+          const createNewUser = async () => {
+            const start = Date.now()
+            logger.info(`Creating new user for index ${i}...`)
```

Comment:
in my experience in working with mad-dog, I don't think that adding the index is very helpful as it only prints out the index of which libs instance in the libs arr is used. I think it's best we revert this